src/features/marketData/marketDataHooks.ts

```diff
@@ -32,7 +32,7 @@ export const useGasPrices = () => {
     fetchEthMarketDataGasPrices().then((anyBlockPrices) => {
       const ethPrice = {
         chain: BridgeChain.ETHC,
-        standard: anyBlockPrices.standard,
+        standard: anyBlockPrices.fast,
```

Comment:
If `fast` ends up having the same problem as `standard` - do you think it's better to possibly default to like 100 instead of ending up with 1?